crowbar_framework/lib/product.rb

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright 2015, SUSE LINUX Products GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class Product
+  def self.is_ses?
+    ::File.exists? '/opt/dell/barclamps/suse-enterprise-storage'
```

Comment:
Hardcoding the path will make this functionality harder to test outside of production environment.

If suse-enterprise-storage is just another barclamp (installed via an RPM and / or barclamp_install script), I think it should be possible to use the [BarclampCatalog class](https://github.com/crowbar/barclamp-crowbar/blob/master/crowbar_framework/lib/barclamp_catalog.rb) instead:

```ruby
BarclampCatalog.barclamps.key?("suse-enterprise-storage")
```